baywatch.install

```diff
@@ -325,3 +325,14 @@ function baywatch_update_8013() {
   $baywatch = new BaywatchOperation();
   $baywatch->enable_tide_ui_restriction();
 }
+
+/**
+ * Excludes files path in shield setting.
+ */
+function baywatch_update_8014() {
```

Comment:
@vincent-gao does it needs to be in the install as well ? if it does then add it in the bay operations please

Reply:
emm , good question @MdNadimHossain ! looking.

Reply:
@MdNadimHossain changed, thanks.

Reply:
I don't get why don't you just call `$baywatch->exclude_files_path();` here too?

Reply:
😬 yeah, I should. Thanks @anthony-malkoun